src/bundle/whitelist.js

```diff
@@ -51,6 +59,11 @@ export function buildWhitelist() {
       entries: j,
       keys: j,
       values: j,
+      prototype: {
+        // We need to prefix __proto__ with ESCAPE so that it doesn't
+        // just change the prototype of this object.
+        ESCAPE__proto__: 'maybeAccessor',
```

Comment:
Why should `Object.prototype.__proto__` be in the Jessie whitelist?

I'm surprised that we omit `Object.prototype.toString`. What does, for example, 
```js
`${({})}`
''+{}
String({})
```
do?

Reply:
> Why should `Object.prototype.__proto__` be in the Jessie whitelist?

I've been testing with a hacked-up version of SES/realms-shim, but ran into intractable problems with code (like shims, removeProperties, makeHardener, and transforms) needing full access to primordials, so I can't excise the non-Jessie properties and methods.  The whitelist approach is a good one but will probably need to be done via a membrane or else we hamstring any SES code that wants to run in the same compartment.

I'll update the whitelist in the interest of doing something that can lead to a membrane, rather than something that can be fed into SES.

> I'm surprised that we omit `Object.prototype.toString`.

Yeah, that's a good point.  I've added it now.

That's all for another day.  It was fun, but I've got bigger fish to fry right now.